packages/core/stories/badge/dot-badge.qa.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { Badge, GridLayout } from "@salt-ds/core";
+import type { Meta, StoryFn } from "@storybook/react";
+import {
+  QAContainer,
+  QAContainerNoStyleInjection,
+  type QAContainerNoStyleInjectionProps,
+  type QAContainerProps,
+} from "docs/components";
+import { allIcons } from "./icon.all";
+
+export default {
+  title: "Core/Badge/Dot Badge QA",
+  component: Badge,
+} as Meta<typeof Badge>;
+
+const AllIcons: StoryFn = () => {
```

Comment:
Do we need this being a QA story which we need to capture by Chromatic? Or should this just be a normal story which we check internally, given Badge should be applied on something "intractable"?

If we need to snapshot dot badge against an icon, just add one single icon to the existing Badge qa

Reply:
added to existing one